.eslintrc.js

```diff
@@ -3,15 +3,16 @@ module.exports = {
   "env": {
     "mocha": true,
     "node": true,
-    "browser": true,
+    "browser": true
```

Comment:
lol, you don't like commas on final lines? My linter typically does

Reply:
No i do like commas on the final lines, im just not sure as to why some did not have it to begin with. Usually it formats on save and fixes stuff like this but oops